stylus/ui-components/selectionbar.styl

```diff
@@ -0,0 +1,58 @@
+slate-grey = #5d6165
+
+$selectionbar
+    .coz-selectionbar
+        position fixed
+        z-index  1000
+        top      0
+        left     0
+        display         flex
+        justify-content center
+        align-items     center
+        width           100%
+        height          em(56px)
+        color white
+        background-color slate-grey
+        font-weight bold
+
+        .coz-selectionbar-separator
+            width            4px
+            height           4px
+            margin           0 .5em 0 2em
+            background-color black
+            border-radius    50%
+
+        button
+            left-padded(50px)
+            border              0
+            color               white
+            background-color    transparent
+            background-position em(24px) center
+            background-repeat   no-repeat
+            line-height         em(48px)
+            font-size(14px)
```

Comment:
Just write `font-size 14px` instead of font-size(14px) it will still use the fonction, don't worry.
Same for left-padded above.